index.js

```diff
@@ -1,3 +1,7 @@
 function sum(a, b) {
     return a + b;
 }
+
+function sub(a, b) {
```

Comment:
Acho que ia ficar mais descritivo, e você?
```suggestion
function subtraction(a, b) {
```